.pre-commit-config.yaml

```diff
@@ -6,18 +6,18 @@ repos:
     - id: go-vet
     - id: go-lint
  - repo: https://github.com/keep-network/pre-commit-hooks.git
-   rev: 63e729f
+   rev: v1.3.0
    hooks:
     - id: check-added-large-files
  - repo: local
    hooks:
-    - id: solidity-lint-js
+    - id: lint-js
```

Comment:
Let's leave the `solidity-` prefix as we're still in the solidity directory.

Reply:
Done in 46324f9